x-pack/plugins/gis/public/components/map/mb/view.js

```diff
@@ -169,11 +176,22 @@ export class MBMapContainer extends React.Component {
     }
 
     clearGoto();
-    this._mbMap.setZoom(goto.zoom);
-    this._mbMap.setCenter({
-      lng: goto.lon,
-      lat: goto.lat
-    });
+
+    if (goto.bounds) {//prioritize zooming to bounds after clicking
+      const lnLatBounds = new mapboxgl.LngLatBounds(
+        new mapboxgl.LngLat(clamp(goto.bounds.min_lon, -180, 180), clamp(goto.bounds.min_lat, -89, 89)),
```

Comment:
Why are these 89 instead of 90? Maybe a commit as to why it is important to clamp to 89 instead of 90

Reply:
This is a bug in Mapboxgl. Fitting on bounds with 90/-90 latitudes throws an error. The poles are undefined in a Mercator projector. Seems like mapbox-gl doesn't handle this all that great.

Reply:
cool - can you put that in a comment then